src/openrct2-ui/UiContext.cpp

```diff
@@ -713,7 +718,25 @@ class UiContext final : public IUiContext
 
     bool SetClipboardText(const utf8* target) override
     {
+#ifndef __EMSCRIPTEN__
         return (SDL_SetClipboardText(target) == 0);
+#else
+        return (
+            MAIN_THREAD_EM_ASM_INT(
+                {
+                    try
+                    {
+                        navigator.clipboard.writeText(UTF8ToString($0));
+                        return 0;
+                    }
+                    catch (e)
+                    {
+                        return -1;
+                    };
+                },
+                gVersionInfoFull)
```

Comment:
Why is there `gVersionInfoFull` here?

Reply:
Good question.. Mustve missed this. This is why we have code reviews